src/ext/resources/gtfsgraphqlapi/schema.graphqls

```diff
@@ -3854,6 +4088,30 @@ type TicketType implements Node {
     zones: [String!]
 }
 
+input TransferPreferences {
+    costPenalty: Int
+    minimumWaitingTime: Duration
+    maximumAdditionalTransfers: Int = 5
+    maximumTransfers: Int = 12
```

Comment:
Should we include the maximumTransfers parameter at all as it is slightly problematic and do these defaults make sense (I copied them from the transmodel API)?